modules/mini-games/Race.lua

```diff
@@ -413,11 +414,24 @@ local kill_biters = Token.register(function(name)
     end
 end)
 
+--- Unstuck yourself
+Commands.new_command('unstuck', 'Unstuck your car from walls')
+:register(function(player)
+    local car = cars[player.name]
+    if not car or not car.valid then
+        player.print("Not in a car!")
+        return Commands.error
+    end
+
+    local new_position = surface[1].find_non_colliding_position('assembling-machine-1', car.position, 5, 0.5)
+    car.teleport(new_position, surface[1])
+end)
+
 --- Respawn the car for a player
 local respawn_car
 respawn_car = Token.register(function(name)
     local player = dead_cars[name].player
-    local position = surface[1].find_non_colliding_position('car', dead_cars[name].position, 5, 0.5)
+    local position = surface[1].find_non_colliding_position('assembling-machine-1', dead_cars[name].position, 5, 0.5)
```

Comment:
mabye just use smt like rocket silo so you dont need the command at all

Reply:
Rocket silo is 9x9.  That would make it fail in narrow sections and is completely unacceptable.  And even so, it's still possible to get pretty stuck driving parallel to a straight section of wall so you still need the command anyhow.

Reply:
The assembling seams like the right choice as its about the same size as a car but without the rotational difference.